src/Cache/Adapter/Redis.php

```diff
@@ -26,11 +27,13 @@ public function __construct(Client $redis)
     /**
      * @param  string  $key
      * @param  int  $ttl time in seconds
+     * @param  string  $hash optional
      * @return mixed
      */
-    public function load(string $key, int $ttl): mixed
+    public function load(string $key, int $ttl, string $hash = ''): mixed
     {
-        $redis_string = $this->redis->get($key);
+        $redis_string = $this->redis->hGet($key, $hash);
```

Comment:
if $hash is empty to we want to use regular get?

Reply:
I dont think so. Regular GET is not very optimal when it comes to purging